### PR TITLE
Remove "Are you sure you want to close this" modal

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
Preview @ http://remove_modal.surge.sh

### CODE REVIEW

File: src/browser/modules/Frame/FrameTitlebar.tsx [hunk 1]
```diff
@@ -163,6 +161,17 @@ function FrameTitlebar(props: FrameTitleBarProps) {
   }, [props.frame.isRerun])
   useEffect(gainFocusCallback, [gainFocusCallback])
 
+  useEffect(() => {
```
Comment (jharris4) [inline src/browser/modules/Frame/FrameTitlebar.tsx:164]: I'm curious, is this related to removing the modal somehow?

Comment (OskarDamkjaer) [reply src/browser/modules/Frame/FrameTitlebar.tsx:164]: No it's related to be being sloppy :/ good you caught it

**Review by jharris4 — APPROVED** [review]
LGTM
